entries/h/humana.com.json

```
@@ -0,0 +1,13 @@
+{
+    "Humana": {
+      "domain": "humana.com",
+      "tfa": [
+        "sms",
+        "email"
+      ],
+      "documentation": "https://www.humana.com/2-step-verification",
+      "categories": [
+        "health"
+      ]
```

Humana seems to only offer insurance for U.S. residents.
```suggestion
      ],
      "regions": [
        "us"
      ]
```